### DISCUSSION TITLE
Welcome to the Render96 texture pack Discussion

### DISCUSSION THREAD

**Discussion by pokeheadroom** [pokeheadroom/RENDER96-HD-TEXTURE-PACK#4] [General]
👋 Welcome!
  We’re using Discussions as a way to more easily manage the feedback on how to improve the project further. We hope to listen to some great ideas

**Comment by ghost** [comment]
does anybody have an upscaled version of the second jolly roger bay painting because on the github theres a photo of the first one but not the second one (also here is a picture of both the paintings for reference: 
![image](https://github.com/pokeheadroom/RENDER96-HD-TEXTURE-PACK/assets/67620395/8fd6e510-cd7c-4410-af3c-de6e681e7765)
